.github/workflows/scheduled_mdlinkcheck.yaml

```diff
@@ -25,13 +25,13 @@ jobs:
         content-filepath: ./.github/ISSUE_TEMPLATE/found-dead-links.md
         labels: Documentation, Bug
 
-    - name: Create Project Card
-      if: ${{ failure() }}
-      uses: peter-evans/create-or-update-project-card@v1
-      with:
-        token: ${{ secrets.BOT_PAT }}
-        project-location: retaildevcrews
-        project-name: WCNP Sprint Board
-        project-number: '12'
-        column-name: 'Triage'
-        issue-number: ${{ steps.ciff.outputs.issue-number }}
+    # - name: Create Project Card
```

Comment:
Is there an associated issue with turning off project card creation?

Reply:
Added it to PR description

Reply:
Thanks for updating. I'm not seeing anything in issue about turning that off, just ensuring automated credentials. I'm assuming issue is out of date, please modify issue to capture changes. Aside from that, everything else looks good to me!